java/java.lsp.server/vscode/src/launcher.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+'use strict';
+
+import { ExtensionContext } from 'vscode';
+import * as fs from 'fs';
+import * as path from 'path';
+import { spawn, ChildProcessByStdio } from 'child_process';
+import * as vscode from 'vscode';
+import { Readable } from 'stream';
+
+export function launch(
+    context: ExtensionContext,
+    jdkHome: string | unknown,
+    ...extraArgs : string[]
+): ChildProcessByStdio<null, Readable, Readable> {
+    function findClusters(): string[] {
+        let clusters = [];
+        for (let e of vscode.extensions.all) {
+            const dir = path.join(e.extensionPath, 'nb-java-lsp-server');
```

Comment:
I have to say I have the `nb-java-lsp-server` name. Couldn't it be simply `netbeans`, @jlahoda?

Reply:
Fine by me.

Reply:
I am trying to rename it to `nbcode`.